packages/ProgressIndicators/src/ProgressBar/index.tsx

```diff
@@ -31,60 +31,65 @@ export const defaultProgressBarProps = {
 };
 
 /** Displays configurable progress bar */
-const ProgressBar = (props: ProgressBarProps) => {
-  const {
-    animate,
-    decimalPoints,
-    value,
-    lineColor,
-    stripped,
-    lineColorThresholds,
-    showLabel,
-    cssClass
-  } = props;
-  let backgroundColor = lineColor;
-  const max = props.max || 100;
-  const min = props.min || 0;
-  let range = max - min;
-  if (range <= 0) {
-    range = 100;
-  }
-  const decimalValue = value / range;
-  const percentValue = decimalValue * 100;
-  const percentValueString = percentValue.toFixed(decimalPoints);
+class ProgressBar extends Component<ProgressBarProps, {}> {
+  public static defaultProps = defaultProgressBarProps;
 
-  // set the line color: if lineColorThresholds is not given; lineColor will be used
-  if (lineColorThresholds) {
-    // sort the color and their thresholds by the threshold value
-    const AscendingThresholds = Object.entries(lineColorThresholds).sort((e1, e2) => e1[1] - e2[1]);
-    // top to bottom check to see which color threshold is matched first by the percentValue
-    AscendingThresholds.forEach(([key, threshold]) => {
-      if (percentValue >= threshold) {
-        backgroundColor = key;
-      }
-    });
+  constructor(props: ProgressBarProps) {
+    super(props);
   }
 
-  return (
-    <div className="progress">
-      <div
-        className={`progress-bar ${stripped ? cssClass : ''}
-        ${animate ? cssClass : ''}`}
-        style={{
-          backgroundColor: `${backgroundColor}`,
-          width: `${percentValueString}%`
-        }}
-        role="progressbar"
-        aria-valuenow={percentValue}
-        aria-valuemin={min}
-        aria-valuemax={max}
-      >
-        {showLabel ? `${percentValue}%` : null}
-      </div>
-    </div>
-  );
-};
+  public render() {
+    const { props } = this;
+    const {
+      animate,
+      decimalPoints,
+      value,
+      lineColor,
+      cssClass,
+      stripped,
+      lineColorThresholds,
+      showLabel
+    } = props;
+    let backgroundColor = lineColor;
+    const max = props.max || 100;
+    const min = props.min || 0;
+    let range = max - min;
+    if (range <= 0) {
+      range = 100;
+    }
+    const decimalValue = value / range;
+    const percentValue = decimalValue * 100;
+    const percentValueString = percentValue.toFixed(decimalPoints);
+
+    // set the line color: if lineColorThresholds is not given; lineColor will be used
+    if (lineColorThresholds) {
+      // sort the color and their thresholds by the threshold value
+      const AscendingThresholds = Object.entries(lineColorThresholds).sort(
```

Comment:
@RayceeM can we change this to be lowercase (`ascendingThresholds`)? Typically uppercase variables like this are either classes or types :)

Reply:
yep sure!